<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCadastroAcesso</class>
 <widget class="QWidget" name="FormCadastroAcesso">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Acesso</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 143, 157);</string>
  </property>
  <widget class="QTableWidget" name="lista">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 202, 206);
border-radius:5px</string>
   </property>
   <column>
    <property name="text">
     <string>Usuários</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 202, 206);
border-radius:5px</string>
   </property>
   <widget class="QLineEdit" name="txt_cad_usuario">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>76</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>8</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista de Usuários</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_lista_usuarios">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>27</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>58</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisa por Nome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cad_acesso">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>105</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:5px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:5px
}
</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Users/Mario Lopez/.designer/backup/icones/icons8-add-25.png</normaloff>../Users/Mario Lopez/.designer/backup/icones/icons8-add-25.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cad_acesso_sair">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>105</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:5px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:5px
}
</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Users/Mario Lopez/.designer/backup/icones/icons-sinal-de-saída-25.png</normaloff>../Users/Mario Lopez/.designer/backup/icones/icons-sinal-de-saída-25.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 143, 157);
border-radius:5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/icon_title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>lista</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cad_acesso_sair</sender>
   <signal>clicked()</signal>
   <receiver>FormCadastroAcesso</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
